src/main/java/edu/iis/powp/adapter/MyAdapter.java

```diff
@@ -9,6 +10,9 @@
  * Plotter adapter to drawer with several bugs.
  */
 public class MyAdapter extends DrawPanelController implements IPlotter {

```

Comment:
PLEASE FIX:
- missing "nieprecyzyjna nazwa klasy."
- Inheritance is not used (no super drawLine calls). Should be removed!